crmsh/bootstrap.py

```diff
@@ -141,10 +142,10 @@ def log_file_fallback():
     """
     return os.path.join(utils.get_tempdir(), "ha-cluster-bootstrap.log")
 
-
 def log(*args):
     global LOG_FILE
     try:
+        Path(os.path.dirname(LOG_FILE)).mkdir(parents=True, exist_ok=True)
```

Comment:
Such a directory should usually be created by Makefile.am or at the `%install` stage in rpm spec file, and listed as one of the %files there, so that it's maintained by the rpm package.